test/gen_stage/broadcast_dispatcher_test.exs

```diff
@@ -51,23 +51,23 @@ defmodule GenStage.BroadcastDispatcherTest do
     expected_subscribers = MapSet.new([pid1])
 
     {:ok, 0, disp} = D.subscribe([], {pid1, ref1}, disp)
-    assert disp == {[{0, pid1, ref1, nil}], 0, expected_subscribers}
+    assert disp == {[{0, pid1, ref1, nil}], 0, 0, expected_subscribers}
 
     {:ok, 10, disp} = D.ask(10, {pid1, ref1}, disp)
-    assert disp == {[{0, pid1, ref1, nil}], 10, expected_subscribers}
+    assert disp == {[{0, pid1, ref1, nil}], 10, 10, expected_subscribers}
 
     expected_subscribers = MapSet.put(expected_subscribers, pid2)
 
     {:ok, 0, disp} = D.subscribe([], {pid2, ref2}, disp)
-    assert disp == {[{0, pid2, ref2, nil}, {10, pid1, ref1, nil}], 0, expected_subscribers}
+    assert disp == {[{0, pid2, ref2, nil}, {10, pid1, ref1, nil}], 0, 10, expected_subscribers}
 
     expected_subscribers = MapSet.delete(expected_subscribers, pid1)
 
     {:ok, 0, disp} = D.cancel({pid1, ref1}, disp)
-    assert disp == {[{0, pid2, ref2, nil}], 0, expected_subscribers}
+    assert disp == {[{0, pid2, ref2, nil}], 0, 10, expected_subscribers}
 
-    {:ok, 10, disp} = D.ask(10, {pid2, ref2}, disp)
-    assert disp == {[{0, pid2, ref2, nil}], 10, expected_subscribers}
+    {:ok, 0, disp} = D.ask(10, {pid2, ref2}, disp)
```

Comment:
Note that here (and also at line 155) I am reverting the IMO incorrect change in assertions in 5d8193ee2.